content/js1/sprints/1/prep/index.md

```diff
@@ -8,4 +8,443 @@ backlog= 'Module-JS1'
+
+We can use the node command to run a JavaScript file.
+A JavaScript file ends with `.js` - this is known as the file extension.
+
+Let’s suppose we have a file `sums.js` like this:
+
+```js title="sums.js"
+10 + 3;
+10 * 3;
+10 / 3;
+```
+
+In this case, we can write the command `node sums.js`.
+
+This command is an instruction to execute the program written inside index.js. Our program consists of 3 lines, each line an expression. So the machine will start:
+
+`10 + 3;` the computer will evaluate this expression  
+`10 * 3;` the computer will evaluate this expression  
+`10 / 3;` the computer will evaluate this expression
+
+{{<note title="Activity" type="activity">}}
+
+Try out the following steps below to check you can run a file with Node:
+
+1. In your terminal, create a new file called `example.js`.
+
+2. Try writing a few expressions in the file like the ones above.
+
+3. Get Node to run this file. ( Don't use the REPL now - there should be a command to execute the whole file.)
+
+{{</note>}}
+
+Once the computer evaluates these expressions the execution of the program is complete. But, we’re left with a problem. With the REPL, the user inputs an expression, the computer evaluates it and then prints the result. It then loops back and prompts the user for another input. But now the computer will go and execute each line sequentially until completion.
+
+So this new problem can be expressed as a question:
+
+> ❓ Problem
+>
+> "How can we check the values of variables in our program when it is being executed?"
+
+### 🖨️ Printing to the terminal
+
+In order to check the values when our program runs, we can make use of a function called `console.log`.  
+`console.log` will allow us to log expressions to the console while our program is executing. This is very useful for complex programs when we need to check the value of expressions in our program.
+
+Let's see how to use `console.log`
+
+In a file called `example.js`, we can write name of the function `console.log`.
+
+```js title="example.js"
+console.log;
+```
+
+However, if we run this file with Node, we won't be able to see anything in the terminal.
+As with `Math.round` we need to use the syntax for calling a function.
+So we can add parentheses after the function name to indicate we're calling this function:
+
+```js title="example.js"
+console.log("hello there!");
+```
+
+We should see the string "hello there!" logged out in the terminal.
+
+### Saving function output
+
+We can use variables to store the output from a function.
+
+Let’s consider the following file, `arithmetic.js`;
+
+```js title="arithmetic.js"
+const result = Math.round(10.3);
+```
+
+When this program is executed, it creates a variable called result and assigns to it **the output of the function**, in this case the rounded number.
+
+So `result` will store the value `10`.
+
+### 🔭 Logging vs function output
+
+Some functions output values that we can use in our program.
```

Comment:
```suggestion
Some functions return values that we can use in our program.
```

I'd like us to avoid using the term "output" when referring to return values - "return" or "evaluate to" are great, but "output" is super ambiguous and I think pretty unclear when we're trying to distinguish what is output to the terminal vs what is "output" by a function

Reply:
I think `return` is the clearest: let's stick with that for now. I don't think pairing `input` and `return` will be much of an issue.